src/test/java/ai/deepcode/javaclient/core/DeepCodeIgnoreInfoHolderTest.java

```diff
@@ -15,12 +15,10 @@
 
 public class DeepCodeIgnoreInfoHolderTest {
 
-  private final File basicIgnoreFile =
-    new File(getClass().getClassLoader().getResource("basic/.dcignore").getPath());
+  private final File basicIgnoreFile = new File("src/test/resources/basic/.dcignore");
```

Comment:
The previous way looks safer imo

Reply:
But didn't work anymore (on my computer).